.github/workflows/workflow.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           - ubuntu-latest
           - windows-latest
         ocaml-version:
+          - 4.04.2
```

Comment:
It would be nice to test all intermediate versions `4.05`, `4.06`, `4.07` as well, but we are going to end up with a rather large test matrix. We could remove, say, `macos-latest` from the test matrix. What do you think?

Reply:
I think it would be good to not remove macOS. As the builds run in parallel it's not a big problem I think (and it's free :))